netbout-web/src/test/java/com/netbout/rest/bout/XeMessageTest.java

```diff
@@ -70,6 +69,7 @@ public String author() {
                     }
                 }
             ).toXembly()
+        // @checkstyle IndentationCheck (1 line)
```

Comment:
@mkordas See two options, one put 12 char indentation to expect violations of future qulice version upgrades or put the class in the puzzle.